observability_api/endpoints/v1/rules.py

```diff
@@ -31,37 +31,40 @@ def _validate_rule_action(rule: Rule) -> None:
     try:
         action = JourneyService.get_action_by_implementation(rule.journey, rule.action)
         action_factory(rule.action, rule.action_args, action)
-    except ActionTemplateRequired:
+    except ActionTemplateRequired as atr:
         LOG.exception("Action '%s' not configured.", rule.action)
-        raise InternalServerError(f"Action '{rule.action}' not configured, contact system administrator.")
-    except (MultipleActionsFound, ActionException):
+        raise InternalServerError(f"Action '{rule.action}' not configured, contact system administrator.") from atr
+    except (MultipleActionsFound, ActionException) as ae:
         LOG.exception("Action '%s' misconfigured.", rule.action)
-        raise InternalServerError(f"Action '{rule.action}' misconfigured, contact system administrator.")
+        raise InternalServerError(f"Action '{rule.action}' misconfigured, contact system administrator.") from ae
 
 
 def _validate_rule_predicate(rule: Rule) -> None:
     """Validate that the rule data compiles."""
     try:
         parsed_rule_data = RuleDataSchema().load(rule.rule_data)
         compile_schema(rule.rule_schema, parsed_rule_data)
-    except InvalidRuleData:
+    except InvalidRuleData as ird:
         LOG.exception("Invalid rule pattern.\nRule data: %s", pformat(rule.rule_data))
-        raise BadRequest("Invalid rule pattern.")
+        raise BadRequest("Invalid rule pattern.") from ird
 
 
 class Rules(BaseEntityView):
     PERMISSION_REQUIREMENTS: tuple[Permission, ...] = ()
 
     @no_body_allowed
     def get(self, journey_id: UUID) -> Response:
-        """Rule LIST
+        """
+        Rule LIST
         ---
         tags: ["Rule"]
         description: List all rules that attached to a journey.
         operationId: ListRules
         security:
           - SAKey: []
-        parameters:
+
+        Parameters
+        ----------
```

Comment:
These docstrings are yaml and this particular change breaks the parser. Can you revert these?

Reply:
Oh lordy ya'll went with yaml docstrings? I pity thee! :smile: